### PR TITLE
Create UI for manage ASPSP records

### PR DESCRIPTION
**Pull request by berezkin88**

### CODE REVIEW

**Comment by berezkin88** [comment]
omg, completely forgot about tests...

File: xs2a-adapter-rest-impl/src/main/java/de/adorsys/xs2a/adapter/controller/ViewController.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Profile("dev")
+@Controller
+public class ViewController {
+
+    private final String V1_ASPSP_MANAGER = "/manager";
```
Comment (spetrychenko) [inline xs2a-adapter-rest-impl/src/main/java/de/adorsys/xs2a/adapter/controller/ViewController.java]: rename `V1_ASPSP_MANAGER ` to `ASPSP_MANAGER_URL`

File: xs2a-adapter-rest-impl/src/main/java/de/adorsys/xs2a/adapter/controller/ViewController.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package de.adorsys.xs2a.adapter.controller;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Profile("dev")
+@Controller
+public class ViewController {
```
Comment (spetrychenko) [inline xs2a-adapter-rest-impl/src/main/java/de/adorsys/xs2a/adapter/controller/ViewController.java]: rename to `AspspManagerController`

File: xs2a-adapter-rest-impl/src/main/java/de/adorsys/xs2a/adapter/controller/ViewController.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    private final String V1_ASPSP_MANAGER = "/manager";
+
+    @GetMapping(V1_ASPSP_MANAGER)
+    public String manage() {
+        return "./index.html";
```
Comment (spetrychenko) [inline xs2a-adapter-rest-impl/src/main/java/de/adorsys/xs2a/adapter/controller/ViewController.java]: return only "index.html"

File: xs2a-adapter-rest-impl/src/main/resources/static/index.html [hunk 4]
```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Repository UI</title>
```
Comment (spetrychenko) [inline xs2a-adapter-rest-impl/src/main/resources/static/index.html]: rename title to `Aspsp Registry Manager`

File: xs2a-adapter-rest-impl/src/main/resources/static/main.js [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+    for (let i = 1, till = (rowCells.length - 1); i < till; i++) {
+        rowCells[i].setAttribute("contenteditable", true);
+    }
+}
+
+function search() {
+    clearTable();
+
+    let data = document.querySelector(".search-form");
+    let url = "http://localhost:8999/v1/aspsps/?";
```
Comment (spetrychenko) [inline xs2a-adapter-rest-impl/src/main/resources/static/main.js]: remove host and port from the URL, make a request directly to "/v1/aspsps/"

File: xs2a-adapter-rest-impl/src/main/resources/static/main.js [hunk 6]
```diff
@@ -0,0 +1,366 @@
...
+        editButton.style.display = "none";
+        updateOrSaveButton.style.display = "inherit";
+        deleteButton.style.display = "inherit";
+        editableCells(e);
+    }
+}
+
+function deleteButton(e) {
+    let uuidCell = e.parentElement.parentElement.cells[0].innerText;
+    let url = "http://localhost:8999/v1/aspsps/" + uuidCell;
```
Comment (spetrychenko) [inline xs2a-adapter-rest-impl/src/main/resources/static/main.js]: remove hardcoded host

File: xs2a-adapter-rest-impl/src/main/resources/static/main.js [hunk 7]
```diff
@@ -0,0 +1,366 @@
...
+}
+
+function updateButton(e) {
+    let editButton = e.parentNode.children[0];
+    let updateOrSaveButton = e.parentNode.children[1];
+    let deleteButton = e.parentNode.children[2];
+
+    let row = e.parentNode.parentNode;
+
+    let uri = "http://localhost:8999/v1/aspsps/";
```
Comment (spetrychenko) [inline xs2a-adapter-rest-impl/src/main/resources/static/main.js]: remove hardcoded host

File: xs2a-adapter-rest-impl/src/main/resources/static/main.js [hunk 8]
```diff
@@ -0,0 +1,366 @@
...
+}
+
+function saveButton(e) {
+    let editButton = e.parentNode.children[0];
+    let updateOrSaveButton = e.parentNode.children[1];
+    let deleteButton = e.parentNode.children[2];
+
+    let row = e.parentNode.parentNode;
+
+    let uri = "http://localhost:8999/v1/aspsps/";
```
Comment (spetrychenko) [inline xs2a-adapter-rest-impl/src/main/resources/static/main.js]: remove hardcoded host

File: xs2a-adapter-rest-impl/src/main/resources/static/index.html [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+                    <i class="material-icons">
+                        add
+                    </i>
+                </button>
+                <div class="mdl-tooltip" data-mdl-for="add">
+                    Add new Aspsp
+                </div>
+            </div>
+            <div class="export-button" id="export">
+                <a href="http://localhost:8999/v1/aspsps/export" download="aspsps.csv">
```
Comment (spetrychenko) [inline xs2a-adapter-rest-impl/src/main/resources/static/index.html]: remove hardcoded host

**Comment by n0rdy** [comment]
I have 1 general question: do we need to have a number of `console.log(..)` statements for different events? I'm ok with the printing errors, but I'm not sure, that it is needed in other cases.
What do you think?

**Pull request merged**